src/state/user/reducer.ts

```diff
@@ -55,6 +57,7 @@ function pairKey(token0Address: string, token1Address: string) {
 }
 
 export const initialState: UserState = {
+  buyFiatClicked: false,
```

Comment:
doesnt this need a migration?

Reply:
we don't have migration yet on web 😅 (we are moving to redux persist soon from your suggestion remember :P) because of the way its being used in SwapHeader, undefined case is handled correctly / has same expected behavior as initial state false.